lib/cli/cli.js

```diff
@@ -1142,6 +1173,21 @@ module.exports.parseCommandLine = function parseCommandLine() {
         'Will add a link on the waterfall page, helping you to compare the HAR. The full path to your compare installation. In the format of https://compare.sitespeed.io/',
       group: 'HTML'
     })
+    .option('html.pageSummaryMetrics', {
+      type: 'array',
+      describe:
+        'Select from the list of metrics to be tested for the given URL(s)',
```

Comment:
Can you squeeze in a full example here? Users usually have a problem understanding the difference between an array/string in the context of the cli, I've seen so many different solutions/tries :)

Reply:
ok, Peter.